cross/unzip/Makefile

```diff
@@ -12,16 +12,16 @@ COMMENT  = UnZip is an extraction utility for archives compressed in .zip format
 LICENSE  = http://infozip.sourceforge.net/license.html
 
 GNU_CONFIGURE = 1
```

Comment:
`GNU_CONFIGURE = 1` is obsolete with `CONFIGURE_TARGET = nop`

Reply:
Good catch, I can indeed remove the `GNU_CONFIGURE`.  Want me to fix the other ones while at it?
```
spksrc@spksrc-debian10:~/makeflags/spksrc$ grep GNU_CONFIGURE */*/Makefile | grep 1 | while read LINE; do FILE=$(echo $LINE | cut -f1 -d:); grep -q "CONFIGURE_TARGET = nop" $FILE && echo $FILE; done
cross/libcap/Makefile
cross/p7zip/Makefile
cross/ps3netsrv/Makefile
cross/redis/Makefile
cross/unzip/Makefile
```

Reply:
A ripgrep version, I could not help myself: 

```bash
$ rg --pcre2 -U -l -g '**/**/Makefile' '^(?=[\s\S]*\bCONFIGURE_TARGET\s*=\s*nop\b)(?=[\s\S]*GNU_CONFIGURE\s*=\s*1)[\s\S]*$'
cross/p7zip/Makefile
cross/unzip/Makefile
cross/ps3netsrv/Makefile
cross/redis/Makefile
cross/libcap/Makefile
```

The regex is thanks to https://stackoverflow.com/questions/469913/regular-expressions-is-there-an-and-operator#469951

Reply:
Personally I would like to see more smaller PRs if possible. IMHO you can leave them

Reply:
agreed, l'll do a different PR to manage that instaed.

Reply:
> A ripgrep version, I could not help myself:

I so much need to learn to play with it :)

Reply:
@th0ma7 If you are interested in diving into alternatives like ripgrep, I recently found my 2nd favourite: [`fd`](https://github.com/sharkdp/fd) an alternative to find and [`xh`](https://github.com/ducaale/xh) alternative httpie. For even more checkout: https://github.com/TaKO8Ki/awesome-alternatives-in-rust

Reply:
done with https://github.com/SynoCommunity/spksrc/pull/4857